uiowa/uiowa
# Feature: Added margin fixes for layouts

Resolves https://github.com/uiowa/uiowa/issues/3445. 

# How to test
`blt ds --site=sandbox.uiowa.edu`
`yarn workspace uids_base build`

- Go to https://sandbox.local.drupal.uiowa.edu/cards-0 and compare against https://sandbox.prod.drupal.uiowa.edu/cards-0. 
- Try to adjust the column layouts for https://sandbox.local.drupal.uiowa.edu/cards-0 by switching the layout percentage and notice if anything breaks.
